src/containers/AccountConnection.jsx

```diff
@@ -262,6 +263,12 @@ class AccountConnection extends Component {
               date={account && account.lastSync}
             /> }
 
+            {!success && account && account.auth && <KonnectorFolder
+              connector={connector}
+              account={account}
+              driveUrl={this.store.driveUrl}
```

Comment:
This was my first solution, but then I realized that the call to the list of the apps would be done on the first display of the account connection form, and we would have to show the "open folder" link only when we have received the result of the "apps" request, which would cause a shift of the form content.

That is why I chose to do the call to the "apps" request at the page loading once and for all.

What do you think?